mailpoet/lib/Subscription/Captcha/CaptchaRenderer.php

```diff
@@ -45,6 +45,14 @@ public function renderAudio($sessionId, $return = false) {
     exit;
   }
 
+  private function determineAudioType(): string {
+    $type = 'audio';
+    if (isset($_SERVER['HTTP_RANGE'])) {
+      $type .= '-range-' . sanitize_text_field(wp_unslash($_SERVER['HTTP_RANGE']));
```

Comment:
Thanky @lysyjan,
we don't do this for some methods like `sanitize_text_field` and `wp_unslash`, because we use the default WordPress coding standards, which would otherwise complain that we do not sanitize those fields.

References:
Usages of `sanitize_text_field`: https://github.com/mailpoet/mailpoet/search?q=sanitize_text_field
Existence of `sanitizeTextField`: none; https://github.com/mailpoet/mailpoet/search?q=sanitizeTextField

Reply:
👍 Thank you for the explanation. I didn't realize that.